### DISCUSSION TITLE
30 seconds of code

### DISCUSSION THREAD

**Discussion by JSHT** [1101-GS4719-C/Python-Community-GS4719-C#655] [優秀專案]
# [30 seconds of code](https://github.com/30-seconds/30-seconds-of-code)
這個專案主要介紹 JavaScript 使用上的小技巧，至今為止收穫了接近十萬顆 Star。同時他們團隊也有進行介紹 python、CSS 的使用小技巧。其以簡易的方式講述各種效果的呈現，讓大家能夠迅速地上手。團隊架設的網站連結為 https://www.30secondsofcode.org/

**Comment by sys511613** [comment]
感謝分享 之後學習JavaScript的時候可以參考

**Comment by liangyinglly** [comment]
這個也太實用了吧 寫專題如果有遇到困難可以從裡面找耶！

**Comment by 110409538** [comment]
剛剛看了一下網站,發現也有關於python的筆記.感覺很實用.

**Comment by HsuChen88** [comment]
這真的很棒
我現在也有在做一點網頁
這些非常有幫助
謝謝分享

**Comment by jcpan821** [comment]
感謝分享~~

**Comment by 110409538** [comment]
感謝分享！

**Comment by Lynn12378** [comment]
加入書籤了!感覺以後可以用到，謝謝樓主分享

**Comment by Jimmy911218** [comment]
襪賽謝謝樓主提供以後可能會用到哀
